### PR TITLE
Display scraped

### PR DESCRIPTION
**Pull request by edag94**
Seth, please review & Resolve 

Need to add styling for components. Error on line 34:8 of getJSX.js (Unexpected token, this is a blocker)

### CODE REVIEW

File: infoTree/getURLfile.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function getURLindex(label){
+    //first find which URL maps to it
+    let targetURL = "";
+    for (int i = 0; i < whereDoesItGo.length; ++i){
```
Comment (AndyCErnst) [inline infoTree/getURLfile.js]: `whereDoesItGo` is no defined. This will be a runtime error

File: infoTree/getURLfile.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        let cur = whereDoesItGo[i];
+        if (cur.label === label)
+            targetURL = cur.value;
+    }
```
Comment (AndyCErnst) [inline infoTree/getURLfile.js]: this first loop could be replaced with 
`const targetURL= whereDoesItGo.find(el => el.label === label)`

File: infoTree/getURLfile.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    //found mapped URL, now check in URLS.js array to find which file to use
+    let URLindex;
+    for (int URLindex = 0; URLindex < leafURLS.length; ++URLindex){
```
Comment (AndyCErnst) [inline infoTree/getURLfile.js]: leafURLS also not defined

File: infoTree/requireJSON.js [hunk 4]
```diff
@@ -0,0 +1,452 @@
+const URL0 = require("./out/URL0.txt");
```
Comment (AndyCErnst) [inline infoTree/requireJSON.js]: lol, brute force. works for now I guess. I assume 
```
function varMapper(index){ return require("./out/URL" + index + ".txt");
```
doesn't work?

Comment (edag94) [reply infoTree/requireJSON.js]: nope require is done at compile time :(

File: infoTree/getURLfile.js (hunk 1 above)
Comment (edag94) [reply infoTree/getURLfile.js]: This is a global in whereDoesItGo.js
I assume I have to require this variable then?

File: infoTree/getURLfile.js (hunk 3 above)
Comment (edag94) [reply infoTree/getURLfile.js]: Also global

File: infoTree/getURLfile.js (hunk 2 above)
Comment (edag94) [reply infoTree/getURLfile.js]: Hugely in favor of this change, pushing update

File: infoTree/getURLfile.js (hunk 1 above)
Comment (edag94) [reply infoTree/getURLfile.js]: Fixed using import in most recent update

File: infoTree/getURLfile.js (hunk 3 above)
Comment (edag94) [reply infoTree/getURLfile.js]: Fixed using import in most recent update

File: infoTree/requireJSON.js (hunk 4 above)
Comment (AndyCErnst) [reply infoTree/requireJSON.js]: did you try 
```
[ require("./out/URL1.txt"), require("./out/URL2.txt") ... ]
```
?
Again, not necessary for our demo but best practice

File: src/components/Link.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    const {text} = this.props;
+
+    return(
+      <Text style={styles.text} onPress={this.openLink}>
+		{text}
+      </Text>
+    );
+  }
+
+  openLink() {
```
Comment (AndyCErnst) [inline src/components/Link.js]: pro tip: if you write this as an arrow function, you don't have to bind it in the constructor

File: src/components/Bold.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text } from 'react-native';
+
+import {standardStyle} from '../assets/styles.js';
+
+class Paragraph extends React.Component {
+  render() {
+    return (
+      <Text style={[standardStyle.Paragraph, this.props.style]}>{this.props.text}</Text>
```
Comment (AndyCErnst) [inline src/components/Bold.js:9]: should be using the bold style right?

File: src/routes/results.js [hunk 7]
```diff
@@ -40,14 +42,7 @@ class Results extends React.Component {
...
             />
-            <LargeText
-            style={styles.headerUpper}
-            text = {"Material: " + label}
-            />
-            <LargeText
-            style={styles.headerUpper}
-            text = {"Probability: " + probability}
-            />
+            getJSX(URLindexToObj(getURLindex(label)))
```
Comment (AndyCErnst) [inline src/routes/results.js]: typically in packaged javascript you need to export then import these to reference them in another file... but if you are able to access them here maybe react native works differently. 
In a real world app it is (almost) always a terrible idea to use global anything.

**Comment by SethDonohue** [comment]
So are we trying to get this incorporated before the demo today?
Has this been testing on a phone and works?

**Review by AndyCErnst — APPROVED** [review]
don't know your scraper logic well but if it works it works

**Review by AndyCErnst — APPROVED** [review]
One idea: we could remove `https://www.seattle.gov/util/MyServices/WhereDoesItGo` from the files and add it when things get rendered.

**Pull request merged**